testnet-client/src/main.rs

```diff
@@ -402,7 +402,7 @@ string that we send you. It should look something like:
                         u64_to_mob_display(fee),
                     );
                     println!(
-                        "balance after paying this bill will be {}.",
+                        "balance after sending this payment will be {}.",
```

Comment:
While you're at it, I think it should be capital B in `Balance`?

Reply:
no, because it continues the line above; ideally we would test for 80 char width though.

Reply:
ah, missed that.